tests/ieee802154_submac/main.c

```diff
@@ -390,11 +416,34 @@ int txtsnd(int argc, char **argv)
     return send(addr, res, len);
 }
 
-static const shell_command_t shell_commands[] = {
-    { "print_addr", "Print IEEE802.15.4 addresses", print_addr },
-    { "txtsnd", "Send IEEE 802.15.4 packet", txtsnd },
-    { NULL, NULL, NULL }
-};
+static int _init(void)
+{
+    mutex_init(&lock);
+
+    submac.cb = &_cb;
+
+    /* Set the Event Notification */
+    submac.dev.cb = _hal_radio_cb;
+
+    ack_timer.callback = _ack_timeout;
+    ack_timer.arg = NULL;
+
+    luid_base(&long_addr, sizeof(long_addr));
+    eui64_set_local(&long_addr);
+    eui64_clear_group(&long_addr);
+    eui_short_from_eui64(&long_addr, &short_addr);
```

Comment:
Cant you use ` luid_netdev_get_eui64`?

Reply:
with `eui_short_from_eui64`

Reply:
this would reintroduce a dependency to netdev

Reply:
Ah that has extra dependencies you might not want for this test, OK.